core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -3899,19 +4353,58 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
             if (memory)
                 linear_mem_size = memory->memory_data_size;
 #endif
-            if (wasm_copy_exception(module, NULL))
+            if (wasm_copy_exception(module, NULL)) {
+#if WASM_ENABLE_EXCE_HANDLING != 0
+                /* the caller raised an exception */
+                char uncaught_exception[128] = { 0 };
+                bool has_exception =
+                    wasm_copy_exception(module, uncaught_exception);
+
+                /* libc_builtin signaled a "exception thrown by stdc++" trap */
+                if (has_exception
+                    && strstr(uncaught_exception, "exception thrown by stdc++")
+                           == 0) {
```

Comment:
Should be `strstr(uncaught_exception, "exception thrown by stdc++")` but not `strstr(uncaught_exception, "exception thrown by stdc++") == 0`, the latter means string is not found, I understand here it is to check whether it is found.